simpleclient_log4j2/pom.xml

```diff
@@ -42,7 +42,7 @@
         <dependency>
             <groupId>org.apache.logging.log4j</groupId>
             <artifactId>log4j-core</artifactId>
-            <version>2.1</version>
+            <version>2.13.1</version>
```

Comment:
We purposefully stick with older versions, so we don't accidentally depend on new features.

Reply:
Understood. Fixed